Sdg23 add getting started page md (#443)
MDAnalysis/MDAnalysis.github.io: pages/getting_started.md

---
layout: page
title: Getting Started 
order: 1
---

> MDAnalysis is a powerful Python library for analyzing molecular dynamics simulations. This page will guide you through quickly installing MDAnalysis, exploring a basic example, and accessing learning resources to get started.

Reviewer: Nice summary.

## Installation Instructions

> For a quick start, check out our [Installation Quick Start][] guide. 

Reviewer: I am nitpicking here but repeating "quick start" seems superfluous. How about
```suggestion
Our [Installation Quick Start][] guide contains instructions to get MDAnalysis running in a few minutes. Try this guide first.
```

For a more thorough installation with advanced options, including dependencies and environment configurations, see the [installation instructions in the User Guide]({{ site.docs.userguide.url }}/stable/installation.html). 

> Once installed, explore this [basic example][] to load a trajectory and perform a simple analysis.

Reviewer: The example should not be under installation. Make it a separate heading – because that's how you advertised in the summary at the top.

I'd actually include the basic example here. It's nice to immediately see how to use MDAnalysis. The syntax-highlighted code is a good illustration!

> [Installation Quick Start]: https://www.mdanalysis.org/pages/installation_quick_start/

Reviewer: Do not use links with full protocol and hostoname inside the site. For this to work in local builds and on GitHub, use something like `{{ site.baseurl }}/pages/installation_quick_start/` ... but check how we are doing it elsewhere

> [basic example]: https://www.mdanalysis.org/pages/basic_example/

Reviewer: change linking

## Learning Resources

For a detailed collection of tutorials, documentation, and videos, please visit the [Learning MDAnalysis page]({{ site.baseurl }}/pages/learning_MDAnalysis).

___

> If you have questions, you can reach out through the [GitHub Discussions]({{ site.mailinglists.discussion.url }}). Happy coding!

Reviewer: discussions is too narrow: just link to https://www.mdanalysis.org/#participating, i.e., `{{ site.baseurl }}/#participating`.

Author: Thanks @orbeckst. As I am redesigning the "home" page, I used the link to the "community" page  (a new PR coming soon) that contains all info about our channels.  We can revisit this once we have all pages refined.

---
layout: page
title: Getting Started 
order: 1
---

> MDAnalysis is a powerful Python library for analyzing molecular dynamics simulations. This page will guide you through quickly installing MDAnalysis, exploring a basic example, and accessing learning resources to get started.

Reviewer 2: ```suggestion
MDAnalysis is a Python library for analyzing molecular dynamics (MD) simulations. This page will guide you through quickly installing MDAnalysis, exploring a basic example, and accessing learning resources to get started.
```

## Installation Instructions

> Our [Installation Quick Start]({{ site.baseurl }}/pages/installation_quick_start) guide contains instructions to get MDAnalysis running in a few minutes. Try this guide first. 

Reviewer 2: ```suggestion
Our [Installation Quick Start]({{ site.baseurl }}/pages/installation_quick_start) guide contains instructions to get MDAnalysis up and running in a few minutes. Try this guide first. 
```
?

For a more thorough installation with advanced options, including dependencies and environment configurations, see the [installation instructions in the User Guide]({{ site.docs.userguide.url }}/stable/installation.html). 

## Basic example

Once MDAnalysis is installed, you can load a trajectory and perform a simple analysis. A typical usage pattern is to iterate through a trajectory and analyze
coordinates for every frame.

In the following example, the end-to-end distance of a protein and the radius of gyration of the backbone atoms are calculated:

{% highlight python %}
import MDAnalysis
from MDAnalysis.tests.datafiles import PSF, DCD   # test trajectory
import numpy.linalg

# load trajectory and topology into a Universe
u = MDAnalysis.Universe(PSF,DCD)  
# from the 4AKE segid, select
# the first atom named N and the last atom named C
nterm = u.select_atoms('segid 4AKE and name N')[0]
cterm = u.select_atoms('segid 4AKE and name C')[-1]
# select the backbone atoms (AtomGroup)
bb = u.select_atoms('protein and backbone') 

> for ts in u.trajectory:     # iterate through all frames
>     r = cterm.position - nterm.position # end-to-end vector from atom positions
>     d = numpy.linalg.norm(r)  # end-to-end distance
>     rgyr = bb.radius_of_gyration()  # method of AtomGroup
>     print("frame = {0}: d = {1} A, Rgyr = {2} A".format(
>           ts.frame, d, rgyr))

Reviewer 2: ```suggestion
for ts in u.trajectory:  # iterate through all frames
    r = cterm.position - nterm.position  # end-to-end vector from atom positions
    d = numpy.linalg.norm(r)  # end-to-end distance
    rgyr = bb.radius_of_gyration()  # method of AtomGroup
    print("frame = {0}: d = {1} A, Rgyr = {2} A".format(
          ts.frame, d, rgyr))
```
